docs/Remote-Access.md

```diff
@@ -52,35 +52,147 @@ In a more simplified format: SOAP for websites, telnet for command line.
-		xmlns:SOAP-ENC="http://schemas.xmlsoap.org/soap/encoding/" 
-		xmlns:xsi="http://www.w3.org/1999/XMLSchema-instance" 
-		xmlns:xsd="http://www.w3.org/1999/XMLSchema" 
-		xmlns:ns1="urn:TC">
-		<SOAP-ENV:Body>
-	 		<ns1:executeCommand>
-	 			<command>server status</command>
-	 		</ns1:executeCommand>
-		</SOAP-ENV:Body>
-	</SOAP-ENV:Envelope>
-
-The response to this command is what you would expect from the terminal.
+Soap works using standard HTTP POST. The entire post payload is XML.
+
+```xml
+<SOAP-ENV:Envelope  
+    xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/" 
+    xmlns:SOAP-ENC="http://schemas.xmlsoap.org/soap/encoding/" 
+    xmlns:xsi="http://www.w3.org/1999/XMLSchema-instance" 
+    xmlns:xsd="http://www.w3.org/1999/XMLSchema" 
+    xmlns:ns1="urn:AC">
+    <SOAP-ENV:Body>
+	<ns1:executeCommand>
+	    <command>server status</command>
+	</ns1:executeCommand>
+    </SOAP-ENV:Body>
+</SOAP-ENV:Envelope>
+```
+
+The response will look like this:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<SOAP-ENV:Envelope
+  xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/"
+  xmlns:SOAP-ENC="http://schemas.xmlsoap.org/soap/encoding/"
+  xmlns:xsi="http://www.w3.org/1999/XMLSchema-instance"
+  xmlns:xsd="http://www.w3.org/1999/XMLSchema"
+  xmlns:ns1="urn:AC">
+  <SOAP-ENV:Body>
+    <ns1:executeCommandResponse>
+      <result>AzerothCore rev. 6f4f0043c2ab+ 2021-05-18 02:16:59 +0200 (master branch) (Win64, RelWithDebInfo)&#xD;
+Connected players: 0. Characters in world: 0.&#xD;
+Connection peak: 0.&#xD;
+Server uptime: 5 second(s).&#xD;
+Update time diff: 10ms, average: 10ms.&#xD;
+</result>
+    </ns1:executeCommandResponse>
+  </SOAP-ENV:Body>
+</SOAP-ENV:Envelope>
+```
+
+Error response looks like this
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<SOAP-ENV:Envelope
+  xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/"
+  xmlns:SOAP-ENC="http://schemas.xmlsoap.org/soap/encoding/"
+  xmlns:xsi="http://www.w3.org/1999/XMLSchema-instance"
+  xmlns:xsd="http://www.w3.org/1999/XMLSchema"
+  xmlns:ns1="urn:AC">
+  <SOAP-ENV:Body>
+    <SOAP-ENV:Fault>
+      <faultcode>SOAP-ENV:Client</faultcode>
+      <faultstring>Error 401: HTTP 401 Unauthorized</faultstring>
+    </SOAP-ENV:Fault>
+  </SOAP-ENV:Body>
+</SOAP-ENV:Envelope>
+```
+
+You need to authenticate by putting the username and password in the URI like this: `http://soapuser:abcd1234@localhost:7878/` (this is also known as "basic auth")
+
+setting request header `Content-Type: application/xml` is not needed. For now.
+
+## Code Examples
+
+<details>
+    <summary>PHP</summary>
+	
+using built-in [SoapClient](https://www.php.net/manual/en/class.soapclient.php)
+
+```php
+$conn = new SoapClient(NULL, array(
+'location' => "http://{{ ip }}:{{ port }}/",
+'uri'      => 'urn:AC',
+'style'    => SOAP_RPC,
+'login'    => 'soapuser',
+'password' => 'abcd1234'
+));
+echo $conn->executeCommand(new SoapParam('server info', 'command'));
+```
```

Comment:
you can mention the following link for PHP and PERL

Reply:
link?

Reply:
Mention? Other than the "Code Examples" section?
Also there's no Perl example?

Reply:
Just mention